packages/api-plugin/src/generators/trpc-api-generator/files/README.md

```diff
@@ -0,0 +1,17 @@
+# Paymaster Proxy Service
```

Comment:
this should be updated in a follow on to remove the `Paymaster` name above and below